assignments/this.js

```diff
@@ -1,26 +1,53 @@
 /* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+ * in your own words. explain the four principle for the "this" keyword below.
+ *
+ * 1. when called simply it points to the window object
+ * 2. you can call it explicit by binding it
+ * 3. called in an object it refers to it
+ * 4. it is used in constructors
+ *
+ * write out a code example of each explanation above
+ */
 
 // Principle 1
-
 // code example for Window Binding
-
+var x = this;
```

Comment:
why did you use a var?    try to stay in the habit of using let or const. But you're code satisfies the challenge/question asked.